include/color_spinor_field_order.h

```diff
@@ -668,17 +684,16 @@ namespace quda {
 #endif
 
         if (block_float) {
-          // only if we have block_float format do we set these (only block_orthogonalize.cu at present)
           norm = static_cast<norm_t *>(const_cast<void *>(field.Norm()));
-          norm_offset = field.NormBytes() / (2 * sizeof(norm_t));
+          norm_offset = field.Bytes() / (2 * sizeof(norm_t));
```

Comment:
Should this be divided by `2 * sizeof(ghostFloat)`? I feel like I've seen this pattern elsewhere, too.

Reply:
Nevermind, misunderstood the purpose of the variable. All good here.